staging/src/k8s.io/client-go/pkg/third_party/forked/golang/json/fields.go

```diff
@@ -1,501 +0,0 @@
-// Copyright 2013 The Go Authors. All rights reserved.
```

Comment:
Dead file.  As packages are made authoritative and unnecessary links severed, packages disappear.  client-go is a lot smaller now than it was before.